src/main/resources/application.yaml

```diff
@@ -11,7 +11,7 @@ management:
     web:
       base-path: "/"
       exposure:
-        include: health, info, prometheus
+        include: health, info
```

Comment:
prometheus can be left there ? it will be used in future I think as PE was working on it

Reply:
it can. but i'm guess there should be some onboarding stuff? iirc this is just a leftover from spring boot template